src/core/app.lua

```diff
@@ -236,6 +236,15 @@ function main (options)
       assert(not done, "You can not have both 'duration' and 'done'")
       done = lib.timer(options.duration * 1e9)
    end
+
+   local breathe = breathe
+   if options.measure_latency or options.measure_latency == nil then
+      local histogram = require('lib.histogram')
```

Comment:
Yeah I think it should be in core. :+1: I think to even remember @lukego longing for a histogram shm module once or twice.